blog post deprecating jaeger component - tracing module (#941)
kyma-project/website: content/blog-posts/2023-01-16-jaeger-deprecation/index.md

---
title: "Jaeger deprecation"
author:
  name: Andreas Thaler, PO @Kyma"
tags:
  - kyma
  - observability
  - tracing

redirectFrom:
  - "/blog/jaeger-deprecation"
---

As part of the Kyma team working on the Observability capabilities, I'd like to let you know that we decided to deprecate the Kyma tracing component bundling the Jaeger backend for trace collection. In this blog post, I'm going to give you our reasoning behind that decision and outline alternatives.

## The Background

> In my recent [blog post](https://kyma-project.io/blog/2022/9/21/observability-strategy/), I outlined the new strategy for the Observability domain in Kyma. To summarize, the shift towards providing enterprise-grade qualities for Kyma modules moved the focus to enable the users to stream telemetry data into their centralized (outer-cluster) observability backends. Hereby, Kyma explicitly decided to support open and easy integration into existing solutions instead of being yet another provider of a specific observability stack in enterprise-grade quality.

Reviewer: ```suggestion
In my recent [blog post](https://kyma-project.io/blog/2022/9/21/observability-strategy/), I outlined the new strategy for the Observability domain in Kyma. Along with the shift towards providing enterprise-grade qualities for Kyma modules, we move the focus on enabling users to stream telemetry data into their centralized (outer-cluster) observability backends. Therefore, Kyma explicitly decided to support open and easy integration into existing solutions instead of being yet another provider of a specific observability stack of enterprise-grade quality.
```

> Now, we'll apply the same principle to the Jaeger component, and with that the tracing component as it is. Kyma's current tracing component is very lightweight and does not meet enterprise-grade qualities. Thus, the current approach does not fit our strategy and will be deprecated.

Reviewer: ```suggestion
Now, we will apply the same principle to the Jaeger component and the tracing component as a whole. Kyma's current tracing component is very lightweight and does not meet enterprise-grade standards. Thus, the current approach does not fit our strategy and will be deprecated.
```
